pkg/virt-api/webhooks/validating-webhook/admitters/instancetype-admitter.go

```diff
@@ -60,6 +97,15 @@ func admitInstancetypeOrPreference(request *admissionv1.AdmissionRequest, resour
 		return resp
 	}
 
+	instancetypeObj, _, err := webhookutils.GetInstanceTypeFromAdmissionRequest(request)
```

Comment:
I think it won't work this way, because `admitInstancetypeOrPreference` can be called on instancetypes and preferences. `webhookutils.GetInstanceTypeFromAdmissionRequest` will fail when called on a preference?

Reply:
Well. For some reason, it worked well for preferences as well, however, I've split the validation for each of these objects to be separate - like any other object we validate.